app/frontend/src/components/form/Step6.vue

```diff
@@ -12,13 +12,20 @@
         <blockquote>{{ confirmationId }}</blockquote>
       </h2>
 
-      <p>Download a copy of this form to send to your local health authority contact below:</p>
+      <p>Download or email a copy of this form to send to your local health authority contact below:</p>
```

Comment:
<p data-v-65e4e2ae=""><h2>Download and Notify</h2>
<ol>
    <li>Download or email yourself a copy of your form submission.</li>  
    <li>Send a copy of your form submission to the relevant health authority</li>
</ol></p>

Reply:
Implemented in next commit